github_pr.py

```diff
@@ -237,8 +255,11 @@ def main():
         github_update_pr(**args)
 
     gh = Github(args['token'])
-    if not args['numberonly'] and not args['noratelimit']:
-        print "Github Rate Limiting: %d remaining of max %d" % (gh.rate_limiting[0], gh.rate_limiting[1])
+    if (('numberonly' in args) and args['numberonly']):
```

Comment:
thinking about it more, I think it makes the most sense to default the values of all the Boolean parameters to False. That way you can get rid of a lot of ```'key' in args```, and it simplifies the Boolean logic here. You are basically specifying a configuration for the application on the command line, and when you don't set a value, you treat it as unset, when you could easily set it because you already know the desired behavior.

Generally, I don't like using ```pass``` unless it is absolutely necessary. Here I think 
```python
    if not (('numberonly' in args) and args['numberonly']):
        if (('noratelimit' in args) and not args['noratelimit']):
            print "Github Rate Limiting: %d remaining of max %d" % (gh.rate_limiting[0], gh.rate_limiting[1])
```
would be more appropriate.